README.md

```diff
@@ -599,10 +601,9 @@ Emitted when GC responds to the `requestmatchDetails` method.
 
 See the [protobuf schema](https://github.com/SteamRE/SteamKit/blob/master/Resources/Protobufs/dota/dota_gcmessages_client.proto#L1571) for `matchDetailsData`'s object structure.
 
-### `matchmakingStatsData` (`waitTimesByGroup`, `searchingPlayersByGroup`, `disabledGroups`, `matchmakingStatsResponse`)
-* `waitTimesByGroup` - Current average matchmaking waiting times, in seconds, per group.
+### `matchmakingStatsData` (`searchingPlayersByGroup`, `disabledGroups`, `matchmakingStatsResponse`)
 * `searchingPlayersByGroup` - Current players searching for matches per group.
-* `disabledGroups` - I don't know how the data is formatted here, I've only observed it to be zero.
+* `disabledGroups` - Bitmask corresponding to groups in `searchingPlayersByGroup`. These values will be 0.
```

Comment:
Instead of saying "These values will be 0" I think it would be better to state "1 bits correspond to disabled groups" or something to that effect. The way it's stated here might be misunderstood as if this value will always be 0.